ninchatsdk/src/main/java/com/ninchat/sdk/NinchatSessionManager.java

```diff
@@ -1278,6 +1278,18 @@ public String getUserName() {
         }
     }
 
+    // Get username or agentname if it is set in configuration
+    public String getName(boolean agentName) {
```

Comment:
Can we have a bit meaningful name here instead of `agentName` ? May be like `isAgentName`, or `isAgent` , or something else that you found relevant during your implementation.

Reply:
The name is `getName` because it handles the screenname for both agent and user. I don't like this either but I couldn't come up with a better way without having 2 separate methods for the same thing and the way `NinchatMessageAdapter` is constructed having a single method is a bit cleaner. I would've named this `getScreenName`or something like that but I don't know what the username/screenname is called here now that `userName` is different from `agentName`.